packages/jira-adapter/src/filters/workflow/workflow_structure_filter.ts

```diff
@@ -36,6 +36,53 @@ const FETCHED_ONLY_INITIAL_POST_FUNCTION = [
   'IssueStoreFunction',
 ]
 
+const PARTS_SEPARATOR = '::'
+
+const TYPE_TO_FROM_MAP = new Map([
+  ['Initial', 'none'],
+  ['Global', 'any status'],
+  ['Circular', 'any status'],
+])
```

Comment:
since the keys here is a closed list, you can use an object and have type safety at compile time
```suggestion
const TYPE_TO_FROM_MAP: Record<Exclude<TransitionType, 'directed>>, string> = {
  Initial: 'none',
  Global: 'any status',
  Circular: 'any status'
}
```
Also, you can remove the `?? ''` in https://github.com/salto-io/salto/pull/4657/files#diff-22c6fad8ca1e0ddd5a61c7c55d3f89c87887e3ead79fb03579f367cec76f8208R81 after this